experimentA/upload_features_rois.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# WARNING: This script will use your current OMERO login session if one
+# exists. If you want to use a different login ensure you logout first.
+
+from glob import glob
+import numpy as np
+import os
+import pandas as pd
+from skimage.io import imread
+
+import omero.clients
+import omero.cli
+from omero.rtypes import (
+    rstring,
+)
+from omero_rois import (
+    mask_from_binary_image,
+    NoMaskFound,
+)
+from omero_upload import upload_ln_s
+
+DRYRUN = False
+SKIP_IF_EXISTING_ROIS = True
+PROJECT_NAME = 'idr0062-blin-nuclearsegmentation/experimentA'
+OMERO_DATA_DIR = '/data/OMERO'
+NAMESPACE = 'openmicroscopy.org/idr/analysis/original'
```

Comment:
This reminds me that in https://github.com/IDR/idr-metadata/pull/365/files#diff-6474da02e8703bf37b47be7fa162c096R83, we decided to go for `idr.openmicroscopy.org/study/xxx`. What would be the impact of updating this to `idr.openmicroscop.org/analysis/original` ? In particular how many other file annotations have we created with this namespace?

Reply:
It's been used for ROIs/features which have been attached below the top level study. This includes
- idr0047 https://idr.openmicroscopy.org/webclient/?show=dataset-4451
- idr0052 https://idr.openmicroscopy.org/webclient/?show=image-5514374

Related, idr0001 has atop-level annotation in the namespace `/test/features/annotations`
- https://idr.openmicroscopy.org/webclient/?show=screen-3

Reply:
Looking at the numbers we are talking about

```
[sbesson@prod67-omeroreadwrite ~]$ /opt/omero/server/OMERO.server/bin/omero hql "select count(f) from FileAnnotation f where f.ns='openmicroscopy.org/idr/analysis/original'"
Using session for public@localhost:4064. Idle timeout: 10 min. Current group: Public
 # | Col1  
---+-------
 0 | 10394 
(1 row)

To see details for object, enter line number.
To move ahead one page, enter 'p'
To re-display list, enter 'r'.
To quit, enter 'q' or just enter.
```

Let us move forward with the same namespace for now to get the study in a publishable form. But opinions welcome regarding the downstream analysis/consumer workflow. /cc @jburel @joshmoore